kolibri/core/assets/src/constants.js

```diff
@@ -40,6 +54,144 @@ export const LearningActivities = {
 };
 
+export const ContentKindsToLearningActivitiesMap = {
+  audio: LearningActivities.LISTEN,
+  document: LearningActivities.READ,
+  exercise: LearningActivities.PRACTICE,
+  html5: LearningActivities.EXPLORE,
+  video: LearningActivities.WATCH,
+  topic: 'folder',
+};
+
+// Resource library categories and subcategoriess
+export const LibraryCategories = {
+  SCHOOL: 'school',
+  BASIC_SKILLS: 'basicSkills',
+  WORK: 'work',
+  DAILY_LIFE: 'dailyLife',
+  FOR_TEACHERS: 'forTeachers',
+};
+
+export const SchoolCategories = {
+  MATHEMATICS: 'mathematics',
+  SCIENCES: 'sciences',
+  LITERATURE: 'literature',
+  SOCIAL_SCIENCES: 'socialSciences',
+  ARTS: 'arts',
+  COMPUTER_SCIENCE: 'computerScience',
+  LANGUAGE_LEARNING: 'languageLearning',
+  HISTORY: 'history',
+};
+
+export const MathematicsSubcategories = {
+  ARITHMETIC: 'arithmetic',
+  ALGEBRA: 'algebra',
+  GEOMETRY: 'geometry',
+  CALCULUS: 'calculus',
+  STATISTICS: 'statistics',
+};
+
+export const SciencesSubcategories = {
+  BIOLOGY: 'biology',
+  CHEMISTRY: 'chemistry',
+  PHYSICS: 'physics',
+  EARTH_SCIENCE: 'earthScience',
+  ASTRONOMY: 'astronomy',
+};
+
+export const LiteratureSubcategories = {
+  LITERATURE: 'literature',
+  READING_COMPREHENSION: 'readingComprehension',
+  WRITING: 'writing',
+  LOGIC_AND_CRITICAL_THINKING: 'logicAndCriticalThinking',
+};
+
+export const SocialSciencesSubcategories = {
+  POLITICAL_SCIENCE: 'politicalScience',
+  SOCIOLOGY: 'sociology',
+  ANTHROPOLOGY: 'anthropology',
+  CIVIC_EDUCATION: 'civicEducation',
+};
+
+export const ArtsSubcategories = {
+  VISUAL_ART: 'visualArt',
+  MUSIC: 'music',
+  DANCE: 'dance',
+  DRAMA: 'drama',
+};
+
+export const ComputerScienceSubcategories = {
+  PROGRAMMING: 'programming',
+  MECHANICAL_ENGINEERING: 'mechanicalEngineering',
+  WEB_DESIGN: 'webDesign',
+};
+
+export const BasicSkillsCategories = {
+  LITERACY: 'literacy',
+  NUMERACY: 'numeracy',
+  DIGITAL_LITERACY: 'digitalLiteracy',
+  LOGIC_AND_CRITICAL_THINKING: 'logicAndCriticalThinking',
+  LEARNING_SKILLS: 'learningSkills',
+};
+
+export const WorkCategories = {
+  PROFESSIONAL_SKILLS: 'professionalSkills',
+  TECHNICAL_AND_VOCATIONAL_TRAINING: 'technicalAndVocationalTraining',
+};
+
+export const VocationalSubcategories = {
+  TOOLS_AND_SOFTWARE_TRAINING: 'toolsAndSoftwareTraining',
+  SKILLS_TRAINING: 'skillsTraining',
+  INDUSTRY_AND_SECTOR_SPECIFIC: 'industryAndSectorSpecific',
+};
+
+export const DailyLifeCategories = {
+  PUBLIC_HEALTH: 'publicHealth',
+  ENTREPRENEURSHIP: 'entrepreneurship',
+  FINANCIAL_LITERACY: 'financialLiteracy',
+  CURRENT_EVENTS: 'currentEvents',
+  ENVIRONMENT: 'environment',
+  MEDIA_LITERACY: 'mediaLiteracy',
+  DIVERSITY: 'diversity',
+  MENTAL_HEALTH: 'mentalHealth',
+};
+
+export const TeachersCategories = {
+  GUIDES: 'guides',
+  LESSON_PLANS: 'lessonPlans',
+};
+
+export const ResourcesNeededTypes = {
+  FOR_BEGINNERS: 'forBeginners',
+  TEACHERS_AND_PEERS: 'toUseWithteachersAndPeers',
```

Comment:
I know this is a constant, and not a string, but shouldn't the `T` in _Teachers_ be capitalized? 🙂